base-xenial/set_local_dev_hostname

```diff
@@ -1,10 +1,18 @@
 #!/bin/bash
 
-docker_for_mac=$(dig +short docker.for.mac.localhost)
+# new internal endpoint for docker for mac
+docker_for_mac=$(dig +short docker.for.mac.host.internal)
+
+# deprecated endpoint (as of 17.12), but still required for older versions of docker
+docker_for_mac_deprecated=$(dig +short docker.for.mac.localhost)
```

Comment:
Will we need to make this change in the service repos as well? Or is this the mechanism that sets up the `/etc/hosts` mappings in all containers?